.github/ISSUE_TEMPLATE/compatibility_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Compatibility request
+about: Suggest supporting a new platform version OpenSearch
+title: 'Compatibility with OpenSearch (version)'
+labels: compatibility
```

Comment:
```suggestion
labels: compatibility, level/task, type/research
```

Reply:
The compatibility tag does not exist in this repo. Import it from the wazuh-kibana-app repo